### ISSUE TITLE
Not working on my macbook air

### ISSUE THREAD

**Issue by mike-symon** [royshil/obs-backgroundremoval#106]
Hi Team, I have extracted the zip in the correct place and I cannot see it in source filters. Am I missing something? Does anyone have any alternatives they know of that I could try? Thanks

**Comment by markrittenhouse** [comment]
I had similar issue on my M1 macbook air, and I found https://github.com/royshil/obs-backgroundremoval/issues/109
You can download release from this fork https://github.com/umireon/obs-backgroundremoval/releases/tag/v0.4.3-umireon and follow instructions, like
```
mkdir -p ~/"Library/Application Support/obs-studio/plugins"
unzip ~/Downloads/obs-backgroundremoval-macosx-arm64.zip -d ~/"Library/Application Support/obs-studio/plugins/"
```

**Comment by duleorlovic** [comment]
I had similar issue on my M1 macbook air, and I found https://github.com/royshil/obs-backgroundremoval/pull/113
You can download release from this fork https://github.com/umireon/obs-backgroundremoval/releases/tag/v0.4.3-umireon and follow instructions, like
```
mkdir -p ~/"Library/Application Support/obs-studio/plugins"
unzip ~/Downloads/obs-backgroundremoval-macosx-arm64.zip -d ~/"Library/Application Support/obs-studio/plugins/"
```

**Comment by mike-symon** [comment]
Thanks mate, will give it a try 👍🏼

**Comment by mrtoner** [comment]
Just a note: the installation instructions for macOS are wrong: you don't simply put the unzipped folder into the OBS package Contents as indicated. There are two folders, Plugins and Resources, within the enclosing folder. The files or folders within those two folders are *moved* into the existing directory structure of Contents. Once that is done, the Background Removal filter will show up.

If the instructions given above, to put the unzipped folder into the Application Support folder, work, that would be *much* easier; however, OBS doesn't see the plugin on my machine.

**Comment by lindsaymarkward** [comment]
Hi @mrtoner and @umireon - Thanks for this info... can you provide a bit more detail on this install process?
For me, the original (0.4.0) zip file contains `Plugins` and `Resources` when extracted, but this https://github.com/umireon/obs-backgroundremoval/releases/tag/v0.4.3-umireon only extracts to one `obs-backgroundremoval.plugin` file. I can "Show Package Contents", but inside this is `MacOS` and `Resources`, not `Pludins`.

Can you provide enough detail to know what to do to get this plugin working please? Thank you so much 🙏

**Comment by umireon** [comment]
@lindsaymarkward 
The plugin installation method was changed from OBS 28. Do not follow the instructions on [the README.md](https://github.com/royshil/obs-backgroundremoval/issues/106) as it won't work.

Follow the below instructions.

```
cd ~/Downloads
curl -LO https://github.com/umireon/obs-backgroundremoval/releases/download/v0.4.3-umireon/obs-backgroundremoval-macosx-arm64.zip
mkdir -p ~/"Library/Application Support/obs-studio/plugins"
unzip obs-backgroundremoval-macosx-arm64.zip -d ~/"Library/Application Support/obs-studio/plugins/"
```

If this doesn't work, please ensure your OBS installation is v28 and Apple Silicon one.

Additional note, do not try to manually unzip and place files.

**Comment by lindsaymarkward** [comment]
Thank you. 
I had tried that but without using curl, only click-to-download (still unzip in Terminal). 
So, I did it with curl then unzip in the terminal. No result.
To be sure, since you said, I reinstalled the Apple Silicon version of OBS from their site. It didn't look any different (About said the same thing), but this time Background Removal was there! Thank you.
(I never got the security prompt, by the way.)
Will you update your release notes instructions? https://github.com/umireon/obs-backgroundremoval/releases/tag/v0.4.3-umireon

**Comment by umireon** [comment]
@lindsaymarkward The instructions were updated for later people, thanks!

<img width="1205" alt="スクリーンショット 2022-10-11 13 18 31" src="https://user-images.githubusercontent.com/1067855/194996154-2c17252c-5013-49da-93a6-5bd7c8ecc5b7.png">

<img width="1007" alt="スクリーンショット 2022-10-11 13 18 16" src="https://user-images.githubusercontent.com/1067855/194996150-a07d2e31-293e-4955-9a24-dce2983374ba.png">

**Comment by twe4ked** [comment]
@umireon any chance of upstreaming your changes?

**Comment by umireon** [comment]
I've released v0.5.8 on my own fork which can be installed with M1 OBS 28+.
https://github.com/umireon/obs-backgroundremoval/releases/tag/v0.5.8

**Comment by royshil** [comment]
M1 now supported.
